Ukol2_01.txt

```diff
@@ -0,0 +1,120 @@
+#names
+
+names_list = ['Jiri', 'Jan', 'Marie', 'Petr', 'Jana', 'Josef',
+'Pavel', 'Martin', 'Jaroslav', 'Tomas', 'Eva', 'Miroslav',
+'Hana', 'Anna', 'Zdenek', 'Frantisek', 'Vaclav', 'Michal',
+'Lenka', 'Katerina']
+
+
+def name_know(name):
+    if name in names_list:
+        print("reply")
+    else:
+        print("another reply")
+print("call 'name_know('your name')'")
+
+name_know(input("Yer name"))
+
+#spelling bee
+
+d = {'a':'alfa', 'b':'bravo', 'c':'charlie', 'd':'delta',
+'e':'echo', 'f':'foxtrot', 'g':'golf', 'h':'hotel', 'i':'india',
+'j':'juliett', 'k':'kilo', 'l':'lima', 'm':'mike',
+'n':'november', 'o':'oscar', 'p':'papa', 'q':'quebec',
+'r':'romeo', 's':'sierra', 't':'tango', 'u':'uniform',
+'v':'victor', 'w':'whiskey', 'x':'x-ray', 'y':'yankee',
+'z':'zulu'}
+
+def name_spell(name):
+    for letter in name:
+        print(letter.lower(), " as in ", d[letter.lower()])
+
+name_spell(input("Yer name"))
+
+#MatrixT
+
+a = [[1,2,3], [4,5,6], [7,8,9]]
+
+def T(Sup_list):
+    new_1 = []
+    new_2 = []
+    new_3 = []
+    for list in Sup_list:
+        new_1.insert(len(new_1), list[0])
+        new_2.insert(len(new_2), list[1])
+        new_3.insert(len(new_3), list[2])
+    fin_a = [new_1,new_2,new_3]
+    return fin_a
+
+print(T(a))
+
+#going shopping
+shopping_list = ["liver", "spleen", "kidney"]
+def new_list(it):
+    new_shopping_list = ["liver", "spleen", "kidney"]
+    for i in shopping_list:
```

Comment:
Pozor! Funkce se odkazuje na promennou mimo funkci (`shopping_list`). Bud byste to mel mit jako vstup do funkce.